test/should.js

```diff
@@ -676,6 +777,49 @@ describe('should', function() {
     }, "blah: expected 1 to have property 'length'");
   });
 
+  it('below(n) (dates)', function(){
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1000);
+    var oneSecondAfter = new Date(now.getTime() + 1000);
+
+    (now).should.be.below(oneSecondAfter);
+    (oneSecondAgo).should.be.lessThan(now);
+    (now).should.not.be.below(oneSecondAgo);
+    (oneSecondAfter).should.not.be.below(oneSecondAgo);
+
+    err(function(){
+      (now).should.be.below(now, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to be below " + now.toUTCString());
+
+    err(function(){
+      (now).should.not.be.below(oneSecondAfter, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to be at least " + oneSecondAfter.toUTCString());
+
+    err(function(){
+      (now).should.have.length.below(3, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to have property 'length'");
+
+    err(function () {
+      (now).should.be.below(null, 'blah');
+    }, "blah: the argument to below must be a date");
+
+    err(function () {
+      (1).should.not.be.below(now, 'blah');
+    }, "blah: the argument to below must be a number");
+
+    err(function () {
+      (now).should.not.be.below(1, 'blah');
+    }, "blah: the argument to below must be a date");
+
+    err(function () {
+      (now).should.not.be.below(null, 'blah');
+    }, "blah: the argument to below must be a date");
+
+    err(function () {
+      ('string').should.have.length.below(now, 'blah');
+    }, "blah: the argument to below must be a number");
+  });
+
```

Comment:
Don't need parenthesis around `<var name>` whenever it's `(<var name>).should`; still need parenthesis for the values plugged directly, like `(0).should`